### DISCUSSION TITLE
please help, can't install

### DISCUSSION THREAD

**Discussion by needhelp212** [SafeExamBrowser/seb-win-refactoring#420] [General]
I am trying to install SEB but keep encountering this error (see attached photo). What can I do to install the program?
![CaptureSEB](https://user-images.githubusercontent.com/106547180/171029518-dbd5121a-3425-4731-9d8e-8f10f40fc657.JPG)

**Comment by dbuechel** [comment]
Could you please post the log file of the setup?
